markdown/chapters/meandeck/homework.md

```diff
@@ -0,0 +1,684 @@
+list. Even following the precepts in the deck construction chapter, this is very
+hard to do in isolation so it's a good idea to find some inspiration first. 
+There are plenty of sites online that publish league lists, high tournament
+finishes and so on. There is also an active community, many who are happy to
+share their current lists and their thoughts/feelings on them. 
+
+Sometimes the metagame will dictate a favourite in the various styles of the 
+deck but a lot of the time there will be no "best list" to run. Simply a large
+variety of ways you could construct your 75 cards. 
+
+### Adapting Online Lists
+
+Once you've found a list you like the look of, it's time for some tuning. Maybe
+you saw something top 8 a Sunday Challenge or perhaps you really liked a brew
+mentioned in a video. Whatever it is, get your deck-building tool of choice and
+start iterating it to your own. 
+
+This doesn't mean you have to make changes to any list you start from but it
+does mean you should really think about each and every slot within it and
+justify to yourself either 1. Why it is worth running or 2. If you would prefer
+to run something else in its place. Maybe the author of the list made some 
+footnotes or is happy to converse on their thoughts? Even so you should just
+look it over anyways. It is preferable to scrutinise any deck you find online
+and try to really understand it.
+
+Some questions you could be asking yourself:
+
+ - What turn does it want to cast Doomsday?
+ - Is it suitable for your target meta?
+ - Does it fit your playstyle?
+ - What cards does it not play and why?
+
+Even if you arrive at the same point as the original list, you will now have a 
+greater understanding of how it is constructed.
+ 
+### Testing Flex Slots
+
+There are a lot of flex slots within Doomsday decks. Sometimes these have low
+impact or play differential (switching a 4th Lotus Petal for a 3rd Cabal Ritual)
+whilst other times they can have significant effects on how you are able play 
+(running vs not running Consider). If you are unsure about any slots, especially
+singleton *pile cards*, one good method is to strip your list right back to
+not have any of the *pile cards* and just playtest some scenarios (using a 
+generic replacement for those cards like Street Wraiths or Preordains). 
+Goldfishing (as will be discussed later) with these limitations will help you
+identify, at least to start with, what tools you feel you truly need and which
+you are happy to be without. 
+
+:::row{variant=CENTERED}
+- DA
+- IU
+- Con
+- LED
+- Pdt
+- PoN
+- NW
+:::
+
+## Methods of Practice
+
+Once you have a decklist drafted (remember, it can always be changed later!), it
+is time to get practicing with it. There are a couple of ways you can do this. 
+Some will require more time than others and, as everyone's lifestyles are
+different, this means some will be more suited to you than others. That's
+totally fine. 
+
+### Gold-Fishing
```

Comment:
Goldfishing is constructed/punctuated 3 different ways in this section. Standardize on one style?